lib/mayaUsd/fileio/utils/xformStack.cpp

```diff
@@ -437,10 +437,11 @@ UsdMayaXformStack::MatchingSubstack(const std::vector<UsdGeomXformOp>& xformops)
 
         if (GetNameMatters()) {
             // First try the fast attrName lookup...
-            const auto foundTokenIdxPairIter
-                = _sharedData->m_attrNamesToIdxs.find(xformOp.GetName());
+            const TfToken& opName = xformOp.GetName();
+            const auto     foundTokenIdxPairIter = _sharedData->m_attrNamesToIdxs.find(opName);
             if (foundTokenIdxPairIter == _sharedData->m_attrNamesToIdxs.end()) {
                 // Couldn't find the xformop in our stack, abort
+                // TF_WARN("Cannot find xform op %s", opName.GetText());
```

Comment:
Note: I left these debug print in comments behind because I found them useful to understand what was going on, so They will be there for teh next dev that need to investigate problem with this complex code.